perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -67,6 +72,21 @@
                      'particular test scenario. Currently only applicable to '
                      'the ListConsistency scenario, ignored in others.')
 
+flags.DEFINE_integer('objects_per_stream', 1000, 'The number of objects to '
```

Comment:
In the big commit I pushed today, I followed your suggestion and added the outline of object size distribution support. Here is how it works at a high level:

- the user supplies a distribution to the benchmark using `--object_storage_object_sizes=...`. The distribution can either be a single size, like "1KB", or a dictionary of sizes and percentages, like "{10KB: 10%, 100KB: 90%}".
- the distribution is passed to `_DistributionToBackendFormat`, which converts it to a slightly simpler format: all object sizes are integers, representing the size in bytes, and the frequencies are floating-point numbers.

It adds complexity to have two different representations of distributions, but I think it's worth it because it lets the user specify distributions using units, but lets us keep pint out of the worker (I especially wanted this because we have to customize pint's unit system slightly, and I don't want to keep the customizations in sync in two places). This also lets us pass distributions to the backend as a JSON data structure, which means we can use the json library that comes with Python to serialize and deserialize, so we don't add any dependencies in the worker.
- The worker script accepts a distribution in the "backend format" and turns that into a Python iterator object, which can be used to draw from the distribution with `next()`.

This is the only part where full distribution support is not complete: the iterator object requires that the distribution has only one size in it. I think this design is going to make it very easy to add full distribution support in a future PR.